chrome/content/zotero/integration/citationDialog/keyboardHandler.mjs

```diff
@@ -86,12 +86,17 @@ export class CitationDialogKeyboardHandler {
 	_handleTopLevelKeydown(event) {
 		let handled = false;
 		let tgt = event.target;
-		let isKeyboardClickable = tgt.classList.contains("keyboard-clickable") || tgt.tagName.includes("button");
-		// Space/Enter will click on a button or keyboard-clickable components
+		let isKeyboardClickable = tgt.classList.contains("keyboard-clickable") || (Zotero.isWin && tgt.tagName.includes("button"));
```

Comment:
`includes()` because there's a namespace prefix? `.localName == 'button'`?

Reply:
Yes, exactly, there is one `xul:button`. And yes, `.localName` is way better!